students/itsabhi/grid.py

```diff
@@ -0,0 +1,27 @@
+def printnewrow(size):
+    plus = "+"
+    minus = "-"
```

Comment:
The variables 'plus' and 'minus' are not used in the printnewrow function so you don't need to assign them here.